9.6/ol-7/rootfs/libpostgresql.sh

```diff
@@ -625,19 +629,25 @@ is_postgresql_running() {
 #########################
 postgresql_master_init_db() {
     local initdb_args=()
-    if [[ -n "${POSTGRESQL_INITDB_ARGS[*]}" ]];then
+    if [[ -n "${POSTGRESQL_INITDB_ARGS[*]}" ]]; then
         initdb_args+=(${POSTGRESQL_INITDB_ARGS[@]})
```

Comment:
Quote to prevent word splitting, or split robustly with mapfile or read -a
```suggestion
        initdb_args+=("${POSTGRESQL_INITDB_ARGS[@]}")
```
https://github.com/koalaman/shellcheck/wiki/SC2206